app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,41 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run, initiator, benchmark_type, benchmark_result_type)
+
+    category = benchmark_run.result.keys[0]
+    current_initiator = benchmark_run.initiator_id
+    last_initiator = BenchmarkRun.where(initiator_type: Commit, benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).order(:created_at).first.initiator_id
+
+    previous_benchmark_runs = BenchmarkRun.where(initiator_type: Commit, benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).offset(last_initiator - current_initiator).limit(1000)
+
+    commit_results = Array.new
+    previous_benchmark_runs.drop(1).each do |run|
+      commit_results << run.result[category].to_f
+    end
+
+    results_avg = commit_results.inject{ |sum, el| sum + el }.to_f / commit_results.size
+    results_standard_deviation = Math.sqrt(commit_results.inject(0){|accum, i| accum +(i-results_avg)**2 }/(commit_results.length - 1).to_f)
+    results_param = results_avg + 2*results_standard_deviation
+
+    if benchmark_run.result[category].to_f > results_param
+      create_issue
+    end
+  end
+
+  def create_issue
+    uri = URI.parse("https://api.github.com/repos/shahsaurabh0605/docker-practice/issues")
+    http_client = Net::HTTP.new(uri.host, uri.port)
+    http_client.use_ssl = true
+    request = Net::HTTP::Post.new(uri.path)
+    request.basic_auth 'RubyBenchBot', 'a12345a'
+    request.body = {'title' => 'Benchmark Regression Detected'}.to_json
+    response = http_client.request(request)
```

Comment:
What should I do if response status is not success? Should I just leave a message or try again?